contents/blog/the-posthog-array-1-18-0.md

```diff
@@ -31,6 +31,17 @@ Following feedback from a user, we have now added support for [PostHog Plugins](
 
 If you're using the chart to deploy PostHog, upgrading to the latest version will give you access to the new plugin server (Beta).
 
+### [Session Recording Improvements](https://github.com/PostHog/posthog/pulls?q=is%3Apr+is%3Aclosed+session)
+
+Out of the many improvements to session recording, there are some worth mentioning specifically:
+
+- Keyboard shortcuts for the session recording player (`spacebar` to pause/play, `spacebar + f` to open in full screen)
```

Comment:
just `f`.

Also left arrow for back by 8 seocnds, right arrow for forward.

Reply:
Go down 2 lines haha